frontend/app/router.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import config from './config/environment';
+
+const Router = Ember.Router.extend({
+  location: config.locationType,
+  rootURL: config.rootURL
+});
+
+Router.map(function() {
+  this.route('signIn');
```

Comment:
Always use `hyphen-case` for route names. 

`sign-in`, `sign-out`, `user-home`, `sign-up` for uniformity with filenames, models, controllers, routes etc.